R/freeze.R

```diff
@@ -9,12 +9,12 @@
 #' \dontrun{
 #' learnR %>% freeze()
 #' }
-freeze <- function(object){
+freeze <- function(object, ...){
```

Comment:
@Cdk29 you forgot to add @param to the documentation part. And please change @return Nothing to None.

Reply:
Indeed, sorry for that.